### PR TITLE
docs: Updated link for CRI-O

### LINKED ISSUE #4767: docs: update link address for CRI-O in docs/install/minikube-installation-guide.md
In file `docs/install/minikube-installation-guide.md`, the link target of CRI-O is:
https://kubernetes.io/docs/setup/minikube/#cri-o
and now it will be redirected to:
https://kubernetes.io/docs/tasks/tools/#cri-o
but this is not the final URL too, the true address should be
https://minikube.sigs.k8s.io/docs/handbook/config/#runtime-configuration

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
In file `docs/install/minikube-installation-guide.md`, updated the link target of CRI-O to https://minikube.sigs.k8s.io/docs/handbook/config/#runtime-configuration
Fixed #4767

### CODE REVIEW

File: docs/install/minikube-installation-guide.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ Here are the features to set up a CRI-O based Minikube, and why you need them:
...
 | what | why |
 | ---- | --- |
-| `--bootstrapper=kubeadm` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+| `--bootstrapper=kubeadm` | As recommended for [minikube CRI-o](https://minikube.sigs.k8s.io/docs/handbook/config/#runtime-configuration) |
 | `--container-runtime=cri-o` | Using CRI-O for Kata |
-| `--enable-default-cni` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+| `--enable-default-cni` | As recommended for [minikube CRI-o](https://minikube.sigs.k8s.io/docs/handbook/config/#runtime-configuration) |
 | `--memory 6144` | Allocate sufficient memory, as Kata Containers default to 1 or 2Gb |
-| `--network-plugin=cni` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+| `--network-plugin=cni` | As recommended for [minikube CRI-o](https://minikube.sigs.k8s.io/docs/handbook/config/#runtime-configuration) |
```
Comment (cmaf) [inline docs/install/minikube-installation-guide.md:62]: s/CRI-o/CRI-O

**Review by cmaf — CHANGES_REQUESTED** [review]
Thanks @PrajwalBorkar, could you please format the commit message like in the [contributing guide](https://github.com/kata-containers/community/blob/main/CONTRIBUTING.md#main-patch) and also make the small update suggested?

Also since @liubin created the issue, I'm wondering where you see the recommendations  for these specific options? I don't see them explicitly in the suggested link, but I do see them on this [page](https://kubernetes.io/de/docs/setup/minikube/#cri-o) in German. However, I can't find the instructions in English...

File: docs/install/minikube-installation-guide.md (hunk 1 above)
Comment (PrajwalBorkar) [reply docs/install/minikube-installation-guide.md:62]: > s/CRI-o/CRI-O

What changes to make? I didn't understand.

Comment (cmaf) [reply docs/install/minikube-installation-guide.md:62]: Sorry for being unclear, could you replace the lowercase o in `[minikube CRI-o]` to uppercase `[minikube CRI-O]`.

**Review by cmaf — CHANGES_REQUESTED** [review]
Thanks for the updates @PrajwalBorkar. The commit still needs a message summarizing the change, and could you please rebase into a single commit?

**Review by cmaf — CHANGES_REQUESTED** [review]
Hi @PrajwalBorkar, could you add a commit message like in the [patch format](https://github.com/intel/intel-device-plugins-for-kubernetes/tree/main/cmd/sgx_plugin#deploying-with-pre-built-images)?

Also it looks like you have a few extra commits. Please follow the instructions for [updating and rebasing](https://github.com/kata-containers/community/blob/main/CONTRIBUTING.md#update-your-pr-based-on-review-comments) the PR to get it to 1 commit.

**Comment by liubin** [comment]
@PrajwalBorkar you need to do a rebase

**Comment by PrajwalBorkar** [comment]
> @PrajwalBorkar you need to do a rebase

Sorry, there are some merge conflicts that occurred after I tried rebasing. Can I open a new clean PR by closing this one?

**Comment by liubin** [comment]
> > @PrajwalBorkar you need to do a rebase
> 
> Sorry, there are some merge conflicts that occurred after I tried rebasing. Can I open a new clean PR by closing this one?

Yes, you can. Fixing git conflicts is really a annoying work.

**Pull request closed without merging**